ssg/templates.py

```diff
@@ -16,21 +16,26 @@
 except ImportError:
     from urllib import quote
 
-languages = ["anaconda", "ansible", "bash", "oval", "puppet", "ignition",
-             "kubernetes", "blueprint", "sce-bash"]
+from collections import namedtuple
+
+templating_lang = namedtuple(
+    "templating_language_attributes",
+    ["name", "file_extension", "template_type", "lang_specific_dir"])
+template_type = ssg.utils.enum("remediation", "check")
+
+languages = [
+    templating_lang("anaconda", ".anaconda", template_type.remediation, "anaconda"),
+    templating_lang("ansible", ".yml", template_type.remediation, "ansible"),
+    templating_lang("bash", ".sh", template_type.remediation, "bash"),
+    templating_lang("blueprint", ".toml", template_type.remediation, "blueprint"),
+    templating_lang("ignition", ".yml", template_type.remediation, "ignition"),
+    templating_lang("kubernetes", ".yml", template_type.remediation, "kubernetes"),
+    templating_lang("oval", ".xml", template_type.check, "oval"),
+    templating_lang("puppet", ".pp", template_type.remediation, "puppet"),
+    templating_lang("sce-bash", ".sh", template_type.remediation, "sce")
+]
```

Comment:
If this was still a map like it was
```python
languages = {
   'anaconda': templating_lang(".anaconda", ...),
   ...
}
```
Then later instead of
```python
lang = [lang for lang in languages if lang.name == supported_lang][0]
```
you could've been able to use plain
```python
lang = languages[supported_lang]
```

Reply:
I am aware of that. However, I thought it would be nice if those language objects were selfcontained... do you think this approach is not good? Maybe I am overengineering.

Reply:
I don't see any benefits in eliminating the map, but traversal is definitely less readable now. Also beware of unguarded index access as well, it will render unhelpful message in case of problems.